reviewlette.gemspec

```diff
@@ -1,11 +1,11 @@
 # coding: utf-8
 lib = File.expand_path('../lib', __FILE__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require 'reviewlette/version'
```

Comment:
@digitaltom Why you removed it? This is a very common way to store the gem verison